vlplidar_ws/scripts/lidar-slam-bringup.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+source $ROS2_WS/install/setup.bash
+
+{
+    ros2 launch vlp_cartographer vlp_driver.launch.py
+} <&0 &
+
+{
+    ros2 launch vlp_cartographer cartographer_demo.launch.py using_odom:=False
+} <&0
```

Comment:
This is copied from my another script which is used to ensure the commands within the brackets can read from standard input. However, it may be unnecessary since we don't input anything to these nodes from standard input. Therefore, I will remove the unused part.